src/settings/accounts-settings.tsx

```diff
@@ -338,6 +361,15 @@ export const AccountsSettings = ({
 					}
 					return {
 						...s,
+						settings: {
+							...s.settings,
+							prefs: {
+								...s.settings.prefs,
+								zimbraPrefDelegatedSendSaveTarget: delegatedSendSaveTarget as NonNullable<
```

Comment:
instead of casting to a nonnullable, we could set empty string as fallback, which is translated to a "reset to default". From the current state of the UI, it should not be possible set this value, but I think that by doing so we can consider it a "covered" case.